packages/slice-machine/components/ScreenshotChangesModal/VariationDropZone.tsx

```diff
@@ -0,0 +1,251 @@
+import { useState } from "react";
+import { Flex, Text, Spinner } from "theme-ui";
+import { useHotkeys } from "react-hotkeys-hook";
+
+import { acceptedImagesTypes } from "@lib/consts";
+import { ToasterType } from "@src/modules/toaster";
+import useSliceMachineActions from "@src/modules/useSliceMachineActions";
+import { AiOutlineCloudUpload } from "react-icons/ai";
+
+import useCustomScreenshot from "./useCustomScreenshot";
+import { useSelector } from "react-redux";
+import { SliceMachineStoreType } from "@src/redux/type";
+import { isLoading } from "@src/modules/loading";
+import { LoadingKeysEnum } from "@src/modules/loading/types";
+import { ScreenshotPreview } from "@components/ScreenshotPreview";
+import { ComponentUI } from "@lib/models/common/ComponentUI";
+
+interface DropZoneProps {
+  imageTypes?: string[];
+  variationID: string;
+  slice: ComponentUI;
+}
+
+const DragActiveView = () => {
+  return (
+    <Flex
+      variant="boxes.centered"
+      sx={{
+        p: 2,
+        position: "absolute",
+        height: "100%",
+        width: "100%",
+        background: "rgba(249, 248, 249, 0.7)",
+        backdropFilter: "blur(5px)",
+        zIndex: "1",
+        pointerEvents: "none",
+      }}
+    >
+      <Flex
+        variant="boxes.centered"
+        sx={{
+          border: "1px dashed #6E56CF",
+          height: "100%",
+          width: "100%",
+          borderRadius: "4px",
+        }}
+      >
+        <UploadIcon isActive />
+        <Text sx={{ my: 2 }}>Drop file here</Text>
+        <Text variant="secondary">Maximum upload size file: 128Mb</Text>
+      </Flex>
+    </Flex>
+  );
+};
+
+export const UploadIcon = ({
+  isActive,
+}: {
+  isActive: boolean;
+}): JSX.Element => (
+  <Flex
+    sx={{
+      p: 1,
+      borderRadius: "50%",
+      bg: isActive ? "#F1EEFE" : "#EDECEE",
+      alignItems: "center",
+      justifyContent: "center",
+      width: "48px",
+      height: "48px",
+    }}
+  >
+    <AiOutlineCloudUpload
+      style={{ color: isActive ? "#6E56CF" : "#6F6E77", fontSize: "34px" }}
+    />
+  </Flex>
+);
+
+const DropZone: React.FC<DropZoneProps> = ({
+  variationID,
+  slice,
+  imageTypes = acceptedImagesTypes,
+}) => {
+  const maybeScreenshot = slice.screenshots[variationID];
+
+  const { openToaster } = useSliceMachineActions();
+  const [isDragActive, setIsDragActive] = useState(false);
+  const [isHover, setIsHover] = useState(false);
+  useHotkeys("meta+v", () => void handlePaste(), []);
```

Comment:
Did you check on Windows / Linux?

Is the meta key this: https://www.w3schools.com/jsref/event_key_metakey.asp#:~:text=On%20Windows%20the%20META%20key,the%20Cmd%20(%E2%8C%98)%20key.

If yes, then on Windows it will be the Window key and it's not how users paste something :/